index.html

```diff
@@ -11,8 +11,11 @@
   <title>Manager</title>
 
   <script type="application/json" id="kuma-config"><%- config %></script>
-
   <script type="module" src="/src/main.ts"></script>
+
+  <link rel="stylesheet" href="/node_modules/@kong/kongponents/dist/style.css">
+  <link rel="stylesheet" href="/node_modules/@kong/kongponents/dist/_variables.scss">
```

Comment:
How come you chose to put these here instead of importing from the CSS?

Reply:
Moved them into main.scss now.